rest/endpoints/user_recordings/profile_recordings.yml

```diff
@@ -265,8 +265,61 @@ paths:
                      code: "30302"
                      title: "Internal Server Error"
                      detail: "Something went wrong while trying to delete the recordings."
+  # ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 
-
+  /user_profiles/{user_profile_id}/recorded_broadcast_slots:
+    get:
+      description: |
+        * **Available since:** 2020-1 (BPLAT-12400)
+      tags:
+        - RecordedBroadcastSlots
+      summary: Fetch all recorded broadcast slots for a given user profile
+      operationId: recordedBroadcastSlots
+      parameters:
+        - in: path
+          name: user_profile_id
+          required: true
+          description: The identity of the user profile to which the recordings belongs.
+          schema:
+            type: string
+      responses:
+        200:
+          description: "Successful response"
+          content:
+            application/json:
+              schema:
+                type: "object"
+                required:
+                  - data
+                properties:
+                  data:
+                    type: array
+                    items:
+                      type: object
+                      required:
+                        - type
+                        - id
+                      properties:
+                        type:
+                          type: string
+                          example: BroadcastSlot
+                        id:
+                          type: string
+                          description: "The id of the entity being referenced"
+                          example: "84eb61a9-75d4-42c7-8c15-84c3d7776227"
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: "https://raw.githubusercontent.com/booxmedialtd/api-documentation/master/rest/snippets/General/JsonApiErrorEntity.yml#/JsonApiErrorEntity"
+              example:
+                errors:
+                  - id: "2d3ba0ea-0aca-11ea-b2d0-4bc6fcad2537"
+                    status: "400"
+                    code: "10100"
+                    title: "Bad Request"
+                    detail: "- is_favourite must be a boolean"
```

Comment:
Weird error example, could you provide something realistic for this endpoint?

Reply:
eww, copy paste, I will update, thx.